src/mono/wasm/test-main.js

```diff
@@ -358,7 +358,38 @@ if (is_browser) {
 Promise.all([argsPromise, loadDotnetPromise]).then(async ([_, createDotnetRuntime]) => {
     applyArguments();
 
-    return createDotnetRuntime(({ MONO, INTERNAL, BINDING, Module }) => ({
+    if (is_node) {
```

Comment:
Please add a brief comment explaining how this is meant to be used, it's not trivial to reverse engineer it from the code